spec/classes/chrony_spec.rb

```diff
@@ -284,7 +284,7 @@
       context 'on any other system' do
```

Comment:
This is a weird place for this test. I realize it was already there before, but it means that for every OS it tests that `on any other system` it fails. It really belongs outside of the `on_supported_os` loop.

Reply:
I think I've successfully moved the test out of the loop in a second commit.